src/resources/js/aggregator-fields.js

```diff
@@ -158,15 +158,16 @@ tribe_aggregator.fields = {
 
 				var importType = $this.val();
 
-				$frequency.select2( 'val', ( 'schedule' === importType ? 'daily' : '' ) ).change();
+				$frequency.val( ( 'schedule' === importType ? 'daily' : '' ) ).change();
 
 				// set a data attribute on the form indicating the schedule type
 				obj.$.form.attr( 'data-type', importType );
 
 				obj.maybeLimitUrlStartDate()
 			} )
 			.on( 'change'     , obj.selector.origin_field              , function() {
-				var origin = $( this ).val();
+				var selectData = $( this ).select2( 'data' );
```

Comment:
I'm not familiar with the methods, but if we're migrating to selectwoo is this `select2` data method OK? do they have the same naming?

Reply:
Yeah we want to avoid moving too much to SelectWoo since we might come back to Select2 in the future if Select2 becomes accessible.